src/data/objects/account.yaml

```diff
@@ -16,10 +27,14 @@ schema:
     type: String
     description: City portion of this account's billing address.
```

Comment:
do we need to say "portion"? Seems weird. Maybe:

The City for this account's billing address.